filters/diag/tarpit.go

```diff
@@ -0,0 +1,63 @@
+package diag
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/zalando/skipper/filters"
+)
+
+type tarpitSpec struct {
+	d time.Duration
```

Comment:
I think this field is unused

Reply:
ah yes it's used as setting the default but never possible to use the default.

Reply:
fixed